spec.html

```diff
@@ -2120,3 +2134,72 @@ <h1>DataView.prototype.setBigUint64 ( _byteOffset_, _value_ [ , _littleEndian_ ]
+    <h1>Agents</h1>
+
+    <p>An <dfn id="agent">agent</dfn> comprises a set of ECMAScript execution contexts, an execution context stack, a running execution context, a set of named job queues, an <dfn id="agent-record">Agent Record</dfn>, and an <dfn id="executing-thread">executing thread</dfn>.  Except for the executing thread, the constituents of an agent belong exclusively to that agent.</p>
+    <p>An agent's executing thread executes the jobs in the agent's job queues on the agent's execution contexts independently of other agents, except that an executing thread may be used as the executing thread by multiple agents, provided none of the agents sharing the thread have an Agent Record whose [[CanBlock]] property is *true*.</p>
+    <emu-note>
+      <p>Some web browsers share a single executing thread across multiple unrelated tabs of a browser window, for example.</p>
+    </emu-note>
+    <p>While an agent's executing thread executes the jobs in the agent's job queues, the agent is the <dfn id="surrounding-agent">surrounding agent</dfn> for the code in those jobs.  The code uses the surrounding agent to access the specification level execution objects held within the agent: the running execution context, the execution context stack, the named job queues, and the Agent Record's fields.</p>
+    <emu-table id="table-agent-record" caption="Agent Record Fields">
+      <table>
+        <tbody>
+          <tr>
+            <th>Field Name</th>
+            <th>Value</th>
+            <th>Meaning</th>
+          </tr>
+          <tr>
+            <td>[[LittleEndian]]</td>
+            <td>Boolean</td>
+            <td>The default value computed for the <em>isLittleEndian</em> parameter when it is needed by the algorithms GetValueFromBuffer and SetValueInBuffer. The choice is implementation-dependent and should be the alternative that is most efficient for the implementation.  Once the value has been observed it cannot change.</td>
+          </tr>
+          <tr>
+            <td>[[CanBlock]]</td>
+            <td>Boolean</td>
+            <td>Determines whether the agent can block or not.</td>
+          </tr>
+          <tr>
+            <td>[[Signifier]]</td>
+            <td>Any globally-unique value</td>
+            <td>Uniquely identifies the agent within its agent cluster.</td>
+          </tr>
+          <tr>
+            <td>[[IsLockFree1]]</td>
+            <td>Boolean</td>
+            <td>*true* if atomic operations on one-byte values are lock-free, *false* otherwise.</td>
+          </tr>
+          <tr>
+            <td>[[IsLockFree2]]</td>
+            <td>Boolean</td>
+            <td>*true* if atomic operations on two-byte values are lock-free, *false* otherwise.</td>
+          </tr>
+          <tr>
+            <td><ins>[[IsLockFree2]]</ins></td>
+            <td><ins>Boolean</ins></td>
+            <td><ins>*true* if atomic operations on two-byte values are lock-free, *false* otherwise.</ins></td>
```

Comment:
8-byte

Reply:
Oops, sorry for these silly mistakes. Fixed.